staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2704,19 +2704,26 @@ type PodSecurityContext struct {
 	// for that container.
 	// +optional
 	RunAsUser *int64 `json:"runAsUser,omitempty" protobuf:"varint,2,opt,name=runAsUser"`
+	// The GID to run the entrypoint of the container process.
+	// Defaults to group specified in image metadata if unspecified.
+	// May also be set in SecurityContext.  If set in both SecurityContext and
+	// PodSecurityContext, the value specified in SecurityContext takes precedence
+	// for that container.
+	// +optional
+	RunAsGroup *int64 `json:"runAsGroup,omitempty" protobuf:"varint,3,opt,name=runAsGroup"`
```

Comment:
have to take an available proto tag at the end of the range... reassigning existing numbers breaks the ability to read in previously persisted data

Reply:
fixing now